src/bci_build/package/appcontainers.py

```diff
@@ -129,6 +129,9 @@
     for os_version in ALL_NONBASE_OS_VERSIONS
 ]
 
+_MARIADB_README = (
+    Path(__file__).parent / "mariadb" / "README.server.md.j2"
+).read_bytes()
```

Comment:
Try this:
```suggestion
).read_text()
```

I suspect that because the readme is read as bytes it is then encoded as utf8 in the `readme` property